specs/agents/logging.md

```diff
@@ -0,0 +1,36 @@
+## Agent logging
+
+### `log_level` configuration
+
+Sets the logging level for the agent.
+
+This option is case-insensitive.
+
+|                |   |
+|----------------|---|
+| Valid options  | `trace`, `debug`, `info`, `warning`, `error`, `critical`, `off` |
+| Default        | `info` |
+| Dynamic        | `true` |
+| Central config | `true` |
+
+### Mapping to native log levels
+
+As not all logging frameworks used by the different agents can natively work with these levels.
+Thus, agents will need to translate them.
+
+Some examples:
+If the logging framework used by an agent doesn't have `trace`,
+it would map it to the same level as `debug`.
+If the underlying logging framework doesn't support `critical`,
+agents can treat that as a synonym for `error`.
+
+The `off` level is a switch to completely turn off logging.
+
+### Backwards compatibility
+
+Most agents have already implemented `log_level`,
```

Comment:
log_level option is available as `logLevel` in RUM and Node.js agent. Do we need to align on the naming? This would be a breaking change.

Reply:
Would it be enough to add an alias?

Reply:
All of the configs follow camel casing in both RUM and Node.js agent, guess thats not a fair thing to do for a single config and forget about others.

Reply:
Yeah, I think alias is the way to go here. All of our other remote config options are lower_case_with_underscores.

Reply:
This is not a proposal to change the casing for options set in code when initializing the agents. An alias is also not required for that.
The spec uses the `lower_case_with_underscores` format as that's the one used in central config. This is the only format that both the RUM agent and the backend agents align on.